glustercli/cmd/volume-reset.go

```diff
@@ -0,0 +1,61 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/gluster/glusterd2/pkg/api"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+const (
+	volumeResetCmdHelpShort = "Reset volume options"
+	volumeResetCmdHelpLong  = "Reset options on a specified gluster volume. Needs a volume name and at least one option"
+)
+
+var (
+	flagForce, flagResetAll bool
+)
+
+var volumeResetCmd = &cobra.Command{
+	Use:   "reset <volname> <options>",
+	Short: volumeResetCmdHelpShort,
+	Long:  volumeResetCmdHelpLong,
+	Args:  cobra.RangeArgs(1, 2),
+	Run: func(cmd *cobra.Command, args []string) {
+		volname := args[0]
+		options := args[1:]
+		req := api.VolOptionResetReq{
+			Force: flagForce,
+			All:   flagResetAll,
+		}
+		if flagResetAll {
+			req.Options = []string{}
+		} else {
+			if len(args) < 2 {
+				failure("Specify atleast one volume option to reset", errors.New("Specify atleast one volume option to reset"), 1)
```

Comment:
@vpandey-RH I think the error message must start with lower case. @Madhu-1 Please confirm

Reply:
@rishubhjain This is CLI. I wouldn't worry too much here.